exporters/zipkin/src/test/java/io/opentelemetry/exporter/zipkin/ZipkinSpanExporterTest.java

```diff
@@ -244,4 +249,55 @@ void stringRepresentation() {
               "ZipkinSpanExporter{endpoint=http://zipkin:9411/api/v2/spans, compressionEnabled=false, readTimeoutMillis=15000}");
     }
   }
+
+  @Test
+  void testSuppressInstrumentation() {
+    TestSpanData testSpanData = spanBuilder().build();
+
+    SuppressCatchingSender suppressCatchingSender = new SuppressCatchingSender(Encoding.JSON);
+    ZipkinSpanExporter zipkinSpanExporter =
+        new ZipkinSpanExporter(
+            new ZipkinSpanExporterBuilder(),
+            mockEncoder,
+            suppressCatchingSender,
+            MeterProvider::noop,
+            mockTransformer);
+
+    byte[] someBytes = new byte[0];
+    Span zipkinSpan =
+        zipkinSpanBuilder(Span.Kind.SERVER, localIp)
+            .putTag(OtelToZipkinSpanTransformer.OTEL_STATUS_CODE, "OK")
+            .build();
+    when(mockTransformer.generateSpan(testSpanData)).thenReturn(zipkinSpan);
+    when(mockEncoder.encode(zipkinSpan)).thenReturn(someBytes);
+
+    zipkinSpanExporter.export(Collections.singleton(testSpanData));
+
+    assertTrue(suppressCatchingSender.sent.get());
```

Comment:
```suggestion
    // Instrumentation should be suppressed on send, to avoid incidental spans related to span export.
    assertTrue(suppressCatchingSender.sent.get());
```